examples/quick-start/add_sub/config.pbtxt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
It won't let me comment on the binary file, so commenting here. Do we really want a 100 MB file in our repo? Is there an easy public place it can be downloaded from?
If not, did you add the file normally? Or did you do some special git magic for large files? There is a way to have large files hosted outside of the normal repo. I forget the term or how it is done, but you should follow up with Elias or Misha.

Reply:
I added the file normally. It is in the example directory - so it is not pulled by when a user downloads our package. Only developers will see this size increase.

Reply:
Talked w/ Misha and Iman. I'm going to find a smaller model to add as an example. The Triton server example directory has a number of small models and I'm hoping one of them produces similar results.

Reply:
Do we not have a place to store these files?

- NGC
- an S3 bucket
- a digitalocean space

This seems like a solid example and hindering it because of 100MB of storage seems counterintuitive to me

Reply:
Would using a Python model be acceptable? They're quite small. For example, there's a Python version of add_sub [here](https://github.com/triton-inference-server/server/tree/main/qa/python_models/add_sub). The parent directory of that folder has a bunch of Python models which are small to store.

I imagine the concern over storage size isn't where to store it as much as that anyone cloning this repo will download the model by default. That'll take more time and use more storage space for this repo. Even if you change the defaults to avoid the MA repo growing in size, if this can be done with a smaller model, it'd probably be preferable for users.

Reply:
Thanks, David. I've switched to using `resnet50_python` for the example. The model.py is much smaller and I was able to get similar results.